### PR TITLE
New batch format - Fixes #11

### LINKED ISSUE #11: Reestructurar el formato del batch
En base a lo que surgió en [este issue](https://github.com/mumuki/mumuki-gobstones-runner/issues/168), se necesita implementar lo charlado en [este PR](https://github.com/gobstones/gobstones-cli/pull/10)

### PR DESCRIPTION
**Pull request by rodri042**
Fixes https://github.com/gobstones/gobstones-cli/issues/11
Related issue: https://github.com/mumuki/mumuki-gobstones-runner/issues/168
Related PR: https://github.com/mumuki/mumuki-gobstones-runner/pull/169

### CODE REVIEW

**Comment by rodri042** [comment]
@luchotc @flbulgarelli

**Comment by luchotc** [comment]
@rodri042 Nice!
Loved the extra validations and the general code cleanup that you introduced :100:

File: README.md [hunk 1]
```diff
@@ -45,23 +45,29 @@ See [the tests](test/run-spec.js) for more examples!
...
+
+```js
+{
+  "code": "procedure Meter(color) { Poner(color) }", // Student's code
+  "extraCode": "function a() { return (2) }", // [OPTIONAL] Teacher's code
+  "examples": [
+    {
+      "initialBoard": "GBB/1.0\nsize 4 4\nhead 0 0", // Initial board
+      "extraBoard": "GBB/1.0\nsize 4 4\nhead 0 0", // [OPTIONAL] Expected board
+      "generatedCode": "procedure Meter(color) { Poner(color) } program { Meter(Azul) }" // [OPTIONAL] Code to be executed, overrides `code`
```
Comment (flbulgarelli) [inline README.md]: Just curious: why are you naming it `generatedCode` and not just `code`?

Comment (rodri042) [reply README.md]: It came from https://github.com/mumuki/mumuki-gobstones-runner/issues/168#issuecomment-494849622 and I think it's a good idea to show that the runner is always generating that code and it's not the student's input.

By the other hand, maybe gobstones-cli shouldn't care about what the runner does and we can leave it as 'code' 

Mixed thoughts on this one. Tell me if you want me to rename it or why do you think  that "code" fits better :)

Comment (flbulgarelli) [reply README.md]: I think it is ok as is. It's difficult to generalize a tool when you have just one use case :stuck_out_tongue:

**Pull request merged**